### PR TITLE
[CORE-1513] Correct project-already-exists error message

### PR DESCRIPTION
**Pull request by robert-uhl**
Turns out that the error message came from the collection package and indicated that the item in the “projects” table already existed.

### CODE REVIEW

File: src/server/pfs/server/driver.go [hunk 1]
```diff
@@ -484,10 +484,18 @@ func (d *driver) createProject(ctx context.Context, req *pfs.CreateProjectReques
 		} else if !errors.Is(err, auth.ErrNotActivated) {
 			return errors.Wrap(err, "could not get caller's username")
 		}
-		return errors.EnsureStack(projects.Create(pfsdb.ProjectKey(req.Project), &pfs.ProjectInfo{
+		if err := projects.Create(pfsdb.ProjectKey(req.Project), &pfs.ProjectInfo{
 			Project:     req.Project,
 			Description: req.Description,
-		}))
+		}); err != nil {
+			if errors.Is(err, col.ErrExists{}) {
```
Comment (albscui) [inline src/server/pfs/server/driver.go:491]: Should this use `errors.As` instead? In the case where a function returns a none default `ErrExists` with extra values filled in, `errors.Is` would return false.
```suggestion
			if errors.As(err, &col.ErrExists{}) {
```

**Review by albscui — COMMENTED** [review]
Maybe add a test?

File: src/server/pfs/server/driver.go (hunk 1 above)
Comment (robert-uhl) [reply src/server/pfs/server/driver.go]: You’re right, although it happens to work due to https://github.com/pachyderm/pachyderm/blob/master/src/internal/collection/errors.go#L50-L53.

Changed.

**Comment by robert-uhl** [comment]
> Maybe add a test?

In this case, it’s just about the English which the user sees, so I figure not such a big deal.

**Review by albscui — APPROVED** [review]
LGTM!

**Pull request merged**